README.md

```diff
@@ -29,14 +29,15 @@ Demonstrate your understanding of this week's concepts by answering the followin
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
 
 1. What is the DOM?
-
+* [ ] Document Object Model is an object representation of the html elements of a webpage. If we want to change the content of page thru JS - we making this not with html directly, we works with DOM.
 2. What is an event?
-
+* [ ] JavaScript's interaction with HTML is handled through events that occur when the user or the browser manipulates a page.
```

Comment:
A user's interaction with the DOM, any input it receives (mouse click/move, keyboard click, etc.)